content/hardware/03.nano/boards/nano-esp32/tutorials/pin-setup/pin-setup.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Nano ESP32 Selecting Pin Configuration 
+description: Learn how to switch between default & ESP32 pin configurations when programming your board.
+author: Karl Söderby
+hardware:
+  - hardware/03.nano/boards/nano-esp32
+tags: [ESP32, Pin Configuration]
+---
+
+The [Arduino Nano ESP32](https://store.arduino.cc/nano-esp32) is a Nano form factor board based on an ESP32-S3 SoC. This board is part of the [Arduino Nano Family](https://store.arduino.cc/pages/nano-family), and follows the pinout of that specific form factor by default.
+
+When programming the board in the "default" mode, the pins labelled on the board matches the pins in the program, **but does not match the ESP32-S3**. This mode can be changed to use the "ESP32" pins instead.
```

Comment:
I think this should clearly mention we are referring to _numbers_ , not _labels_.